spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/RNN.scala

```diff
@@ -293,6 +293,12 @@ class RNN(
       updateOutputTensors = buffer.toArray
     }
 
+    if (isTraining()) {
+      weight.sync()
+      weight_i.sync()
```

Comment:
what does `weight_i` mean ?

Reply:
dnn lstm has two weights, one is weight, and another is iteration weight.